src/main/java/com/chessmates/utility/LichessApiImpl.groovy

```diff
@@ -78,6 +77,7 @@ class LichessApiImpl implements LichessApi {
 
         def pageResults = resultObjects.stream()
                 .map(parse)
+                .filter {item -> item != null }
```

Comment:
Let's go with exceptions here - love the idea of using optionals to make nulls explicit